src/app/migrations/index.js

```diff
@@ -53,9 +53,10 @@ module.exports = function(done) {
     if (err) {
       done(err);
     }
+
     const currentVersion = pkg.version;
     debug('renderer process migrations from %s to %s', previousVersion, currentVersion);
-    if (semver.eq(previousVersion, currentVersion)) {
+    if (semver.gte(previousVersion, currentVersion)) {
```

Comment:
I would put the "placeholder" check in here and skip if we are running with placeholder.

Reply:
Is there a reason why we may want to run migrations in case the previous version is more recent?

Also why the last migration run is not registered rather than relying on the compass version?

Reply:
Well if there is a migration it's not going to work anymore (maybe probably?)? That's what the `migrate` command would check and then fail. Checkout the source of `app-migrations` - it will fail if you try a downgrade and there exists a migration between your lower current version and the higher previous version (which means the migration has already been executed).